### PR TITLE
Add documentation for Sentry

### PR DESCRIPTION
**Pull request by dcramer**
See also home-assistant/home-assistant#30422

### CODE REVIEW

**Review by balloob — DISMISSED** [review]

File: source/_integrations/sentry.markdown [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+## Configuration
+
+To use the `sentry` integration in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+sentry:
+  dsn: SENTRY_DSN
+  environment: development # optional
```
Comment (klaasnicolaas) [inline source/_integrations/sentry.markdown]: We only want required config variables in the first example, so please remove `environment`

File: source/_integrations/sentry.markdown [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
```
Comment (klaasnicolaas) [inline source/_integrations/sentry.markdown:1]: missing: `ha_release`, `ha_category` and the `ha_iot_class`

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_integrations/sentry.markdown (hunk 2 above)
Comment (dcramer) [reply source/_integrations/sentry.markdown:1]: Its unclear to me what ha_category (same w/ ha_iot_class) should be, and given ha_release doesnt yet exist, how do I identify that?

Comment (dcramer) [reply source/_integrations/sentry.markdown:1]: "Other" or a new category for it? There's nothing like "Monitoring" or "Logging" afaict

Comment (dcramer) [reply source/_integrations/sentry.markdown:1]: also found my answer to ha_iot_class:

ha_iot_class: Cloud Polling

Comment (klaasnicolaas) [reply source/_integrations/sentry.markdown:1]: Maybe `System Monitor` is appropriate for this integration. For the release version you can use 0.104, prior to a merge we will check if this version is correct and if not we will adjust it.

File: source/_integrations/sentry.markdown (hunk 1 above)
Comment (dcramer) [reply source/_integrations/sentry.markdown]: Sorry thought I fixed this

**Review by frenck — APPROVED** [review]
Looks great @dcramer! Nice first-time addition! 🥇 

Happy New Year 🍾

**Pull request merged**